### DISCUSSION TITLE
Request to have %MM to use full month name in search template

### DISCUSSION THREAD

**Discussion by sebzapata** [pymedusa/Medusa#10559] [Ideas & Feature requests]
When creating a search template, it would be great to have the option to search by full month name.
For example, a show I like to download has the torrents listed as 'TV Show 8 May 2022' and I am unable to search for them.

Thanks

**Comment by p0psicles** [comment]
Sure

**Reply by sebzapata** [reply]
Hi,
Just wanted to see how this was getting on.

Thanks

**Comment by sebzapata** [comment]
Any update on this?

Thanks

**Comment by Mika3578** [comment]
I have a similar request to the one posted in this discussion, but I also need support for the day in two decimal places, i.e., %Y, %MM,%DD. Can you confirm if the custom template feature is working well for you? Are the requests appearing properly in the Medusa logs?
